Childrens-Social-Care-CPD/Models/RichText.cs

```diff
@@ -16,6 +16,7 @@ public class RichText
         public bool InsetText { get; set; }
 
         public int SortOrder { get; set; }
+        public bool IsAccordion { get; set; }
```

Comment:
Why do we have to detail whether something is an accordian? I would want the presentation to only know how to render this. Am I right that the model in CMS has this property?

Reply:
@rgparkins We want user to set the accordion from Contentful, so we added extra property on Rich text. Without any property how would you know that you need accordion for any section? also we dont want to limit this for any specific page, it should work for any page, not only for Role details

Reply:
@rgparkins We are reusing Richtext for accordion